ui/src/components/flows/blueprints/Blueprints.vue

```diff
@@ -103,7 +104,13 @@
             dotSeparatedTags(tagIds) {
                 return tagIds.map(id => this.tags[id].name).join(".")
             },
-            goToDetail(blueprintId) {
+            goToDetail(clickEvent, blueprintId) {
```

Comment:
We should have native link only, we should not decided for the user to open or not a blank and let them decide with a normal link